lib/tune.cpp

```diff
@@ -966,13 +964,14 @@ namespace quda
           qudaDeviceSynchronize();
           tunable.checkLaunchParam(param);
           if (verbosity >= QUDA_DEBUG_VERBOSE) {
-            printfQuda("About to call tunable.apply block=(%d,%d,%d) grid=(%d,%d,%d) shared_bytes=%d aux=(%d,%d,%d)\n",
+            printfQuda("About to call tunable.apply block=(%d,%d,%d) grid=(%d,%d,%d) shared_bytes=%d aux=(%d,%d,%d,%d)\n",
                        static_cast<int>(param.block.x), static_cast<int>(param.block.y),
                        static_cast<int>(param.block.z), static_cast<int>(param.grid.x), static_cast<int>(param.grid.y),
                        static_cast<int>(param.grid.z), static_cast<int>(param.shared_bytes),
-                       static_cast<int>(param.aux.x), static_cast<int>(param.aux.y), static_cast<int>(param.aux.z));
+                       static_cast<int>(param.aux.x), static_cast<int>(param.aux.y), static_cast<int>(param.aux.z), static_cast<int>(param.aux.w));
           }
 
+          tunable.apply(stream);
```

Comment:
why is this needed?

Reply:
I made a comment on this in [c8a53f7](https://github.com/lattice/quda/pull/1292/commits/c8a53f7d6482cf91ad39b1a1dcf99047cccab7a1).  Without this warmup call, even with locked clocks there were some warnings about second-stage regressions.  This additional call costs nothing, but made the warnings go away.

Reply:
might be worth adding a comment in the code as well.

Reply:
Fair point.  I'll add it in my next PR, whatever that may be....

Reply:
Hopefully one that fixes #1287.....